common/lib/xmodule/xmodule/js/src/capa/display.coffee

```diff
@@ -819,6 +834,6 @@ class @Problem
         MathJax.Hub
         hint_container[0]
       ]
-      hint_container.attr('hint_index', response.hint_index)
+      hint_container.addClass('hint-shown').attr('hint_index', response.hint_index)
```

Comment:
Let's not make any hint changes here. The hint story is https://openedx.atlassian.net/browse/TNL-4934